qiskit/circuit/library/generalized_gates/gms.py

```diff
@@ -91,31 +89,3 @@ def __init__(self, num_qubits: int, theta: Union[List[List[float]], np.ndarray])
             for j in range(i + 1, self.num_qubits):
                 gms.append(RXXGate(theta[i][j]), [i, j])
         self.append(gms.to_gate(), self.qubits)
-
-
-class MSGate(Gate):
-    """MSGate has been deprecated.
-    Please use ``GMS`` in ``qiskit.circuit.generalized_gates`` instead.
-
-    Global Mølmer–Sørensen gate.
-
-    The Mølmer–Sørensen gate is native to ion-trap systems. The global MS can be
-    applied to multiple ions to entangle multiple qubits simultaneously.
-
-    In the two-qubit case, this is equivalent to an XX(theta) interaction,
-    and is thus reduced to the RXXGate.
-    """
-
-    def __init__(self, num_qubits, theta, label=None):
-        """Create new MS gate."""
-        super().__init__("ms", num_qubits, [theta], label=label)
-
-    def _define(self):
-        theta = self.params[0]
-        q = QuantumRegister(self.num_qubits, "q")
-        qc = QuantumCircuit(q, name=self.name)
-        for i in range(self.num_qubits):
-            for j in range(i + 1, self.num_qubits):
-                qc._append(RXXGate(theta), [q[i], q[j]], [])
-
-        self.definition = qc
```

Comment:
This doesn't appear to raise a deprecation warning anywhere.  If it hasn't been warning on usage, it's not eligible for removal yet - we need to put the warnings in, and start the cycle now.

Same goes for the `ms` method of `QuantumCircuit`, which isn't even documented as deprecated.